test/e2e/case/native-protocols/e2e.yaml

```diff
@@ -0,0 +1,147 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is used to show how to write configuration files and can be used to test.
+
+setup:
+  env: compose
+  file: docker-compose.yml
+  timeout: 1200
+  steps:
+    - name: install yq
+      command: |
+        if ! type "yq" > /dev/null; then
+          mkdir -p /tmp/skywalking-infra-e2e/bin && cd /tmp/skywalking-infra-e2e
+          mkdir yq && cd yq
+          wget -O yq.tar.gz https://github.com/mikefarah/yq/archive/v4.11.1.tar.gz
+          tar -zxf yq.tar.gz --strip=1
+          go install && go build -ldflags -s && mv yq ../bin/yq
+          export PATH="$PATH:/tmp/skywalking-infra-e2e/bin"
+          echo "success to install yq"
+        fi
+    - name: install etcdctl
+      command: |
+        if ! type "etcdctl" > /dev/null; then
+          mkdir -p /tmp/skywalking-infra-e2e/bin && cd /tmp/skywalking-infra-e2e
+          mkdir etcdctl && cd etcdctl
+          utype=$(uname | awk '{print tolower($0)}')
+          suffix=
+          if [ $utype = "darwin" ]
+          then
+              suffix="zip"
+          else
+              suffix="tar.gz"
+          fi
+          wget -O etcdctl.$suffix https://github.com/coreos/etcd/releases/download/v3.5.0/etcd-v3.5.0-$utype-amd64.$suffix
+          tar -zxf etcdctl.$suffix --strip=1
+          cp etcdctl ../bin/
+          export PATH="$PATH:/tmp/skywalking-infra-e2e/bin"
+          echo "success to install etcdctl"
+        fi
+
+cleanup:
+  # always never success failure
+  on: always
+
+trigger:
+  action: http
+  interval: 3s
+  times: 5
+  url: http://127.0.0.1:${consumer_9090}/info
+  method: POST
+
+verify:
+  # verify with retry strategy
+  retry:
+    # max retry count
+    count: 20
+    # the interval between two retries, in millisecond.
+    interval: 10000
+  cases:
```

Comment:
`s/127.0.0.1/localhost/g` or `s/localhost/127.0.0.1/g`, let's unify the base url